### PR TITLE
Change map styles to "Voyager"

### PR DESCRIPTION
**Pull request by bramkragten**
## Proposed change

CleanShot 2022-07-19 at 16 42 50

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Review by zsarnett — DISMISSED** [review]

**Comment by noxhirsch** [comment]
Would you consider adding the css filter variables? https://github.com/home-assistant/frontend/pull/13200/commits/7b7230971161f050a9e0b1a3f657b4a8f2c5a06f
While I really like the colored light mode - the colored dark mode map is IMHO way uglier than before. This variable would allow to desaturate the dark map.

**Comment by bramkragten** [comment]
@matthiasdebaat is the men of the styles 🤷‍♂️

**Comment by bramkragten** [comment]
Made it a variable, for example:
With `invert(1) grayscale(1)` you get:

CleanShot 2022-07-19 at 17 16 01

**Review by zsarnett — APPROVED** [review]

**Comment by mltiede** [comment]
How does it look on a PHONE so that it can be viewed outside?  I assume the examples here were on a tablet or desktop.  Google maps seem to be okay to view on phone even outside.  I would say do whatever it takes for the current map system to LOOK like the Google one.  

Screenshot_20220722-103923_Home Assistant

By looking at this map, I have no idea where she is and can barely see the roads.

**Comment by KTibow** [comment]
You can wait for the release to come out, when you can see it for yourself. If there isn't enough contrast, you can adjust map-filter.

**Comment by mltiede** [comment]
Will do.  Thanks.

**Comment by KTibow** [comment]
The map-filter is set to a default, but you can adjust it with a theme.

**Comment by ildar170975** [comment]
> We could also change the built-in filter styling for the dark mode map.

Sorry, what is a "built-in" filter?
The [map card](https://www.home-assistant.io/dashboards/map/) does not gave any filter options. Will they be added?

[Using themes variables](https://github.com/home-assistant/frontend/commit/7b7230971161f050a9e0b1a3f657b4a8f2c5a06f) could be a good variant - at least for people who uses custom themes.
For default themes - well, only hypothetical "map card filter options" will be a solution.

**Comment by KTibow** [comment]
Check the diff, what I'm saying might make more sense then.
TL;DR: Now instead of using 2 seperate URLs for light and dark maps, we now use Voyager. This change applies throughout Home Assistant, everywhere maps are used. No UI options were added.
We also now apply a [`filter`](https://devdocs.io/css/filter) from `var(--map-filter)`. In dark mode, the default is one that makes the light map dark. But you can override it by setting `map-filter: [css filter]` in your theme.

**Comment by Mariusthvdb** [comment]
are we understanding this correctly, that for showing the dark background map in the new frontend, we need to apply extra settings (either in Map config or in themes) than `dark_mode: true `?

Because in latest dev nightly 2022.8.0.dev20220727 with frontend Frontend 20220727.0.dev - latest that still shows as:

Schermafbeelding 2022-07-27 om 14 06 08

**Comment by KTibow** [comment]
It's supposed to be set in the default dark theme, see the diff.

**Comment by Mariusthvdb** [comment]
That’s really awkward, if not a plain regression.
Not using dark_mode in themes at all, users should simply be able to set `dark_mode: true` on map configs, just like the documentation instructs.

**Comment by zsarnett** [comment]
Please do not use merged PRs to discuss issues or the feature. Create an issue or discussion please
